osm2streets/src/render/lane_markings.rs

```diff
@@ -269,6 +286,78 @@ fn draw_parking_lines(lane: &LaneSpec, center: &PolyLine, streets: &StreetNetwor
     result
 }
 
+fn draw_diagonal_parking_lines(
```

Comment:
The markings for the two new cases could be improved to look better, and this code is quite messy. Just getting something started.